doc/reboot-windows.md

```diff
@@ -0,0 +1,35 @@
+# Reboot windows
+The CLUO `update-operator` can be configured to only reboot nodes during certain timeframes.
+
+## Configuring update-operator
+
+The reboot window is configured through the the flags `--reboot-window-start` 
+and `--reboot-window-length`.
```

Comment:
we should probably mention the environment variables that can be set to configure these as well (they come for free from the flag definition, see https://github.com/coreos/container-linux-update-operator/blob/master/cmd/update-operator/main.go#L37)